modules/rosa-install-uninstall-addon.adoc

```diff
@@ -49,14 +49,19 @@ $ rosa install addon --cluster=<cluster_name> | <cluster_id> [arguments]
 |Automatically answers `yes` to confirm the operation.
 |===
 
-.Examples
+.Example
 Add the `codeready-workspaces` add-on installation to a cluster named `mycluster`.
 
 [source,terminal]
 ----
 $ rosa install addon --cluster=mycluster codeready-workspaces
 ----
 
+[NOTE]
+====
+After installing Red Hat CodeReady Workspace, it can be deployed to any namespace except `openshift-workspaces`. For more information, see the link:https://access.redhat.com/documentation/en-us/red_hat_codeready_workspaces/2.10/html/installation_guide/installing-codeready-workspaces_crw#creating-a-project-in-openshift-web-console_crw[Installing the Red Hat CodeReady Workspaces Operator].
```

Comment:
```suggestion
After installing Red Hat CodeReady Workspace, it can be deployed to any namespace except `openshift-workspaces`. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_codeready_workspaces/2.10/html/installation_guide/installing-codeready-workspaces_crw#creating-a-project-in-openshift-web-console_crw[Installing the Red Hat CodeReady Workspaces Operator].
```